Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM thelounge/thelounge:3.0.0
+
+COPY config.js /var/opt/thelounge/
```

Comment:
I am not a docker guru, but I believe it would better to use volumes/mounts (the -v flag) instead of making our own Dockerfile and having to build that every time. That would also allow us to simply restart the container after editing the config, instead of needing to also rebuild the container.

Reply:
In Marathon, the config file was put in secrets (which is currently the only mechanism we have for mounting configurations in containers). I think it's better to build it in the Docker image, since it makes the content of the configuration file clear in our code.

Reply:
If we are going to use mount, then there is no need for a new image. We can use `thelounge`'s official build.